dev/io.openliberty.io.opentelemetry.internal/bnd.bnd

```diff
@@ -59,29 +56,29 @@ Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=11))"
-	io.opentelemetry:opentelemetry-exporter-jaeger;version='1.19.0',\
-	io.opentelemetry:opentelemetry-exporter-logging;version='1.19.0',\
-	io.opentelemetry:opentelemetry-exporter-otlp-common;version='1.19.0',\
-	io.opentelemetry:opentelemetry-exporter-otlp;version='1.19.0',\
-	io.opentelemetry:opentelemetry-exporter-common;version='1.19.0',\
-	io.opentelemetry:opentelemetry-exporter-zipkin;version='1.19.0',\
-	io.opentelemetry:opentelemetry-extension-trace-propagators;version='1.19.0',\
-	io.opentelemetry:opentelemetry-sdk;version='1.19.0',\
-	io.opentelemetry:opentelemetry-sdk-common;version='1.19.0',\
-	io.opentelemetry:opentelemetry-sdk-extension-autoconfigure-spi;version='1.19.0',\
-	io.opentelemetry:opentelemetry-sdk-extension-autoconfigure;version='1.19.0.alpha',\
-	io.opentelemetry:opentelemetry-sdk-logs;version='1.19.0.alpha',\
-	io.opentelemetry:opentelemetry-sdk-metrics;version='1.19.0',\
-	io.opentelemetry:opentelemetry-sdk-trace;version='1.19.0',\
-	io.opentelemetry:opentelemetry-semconv;version='1.19.0.alpha',\
+	io.opentelemetry.instrumentation:opentelemetry-instrumentation-annotations-support;version='1.19.0.alpha';strategy=exact,\
+	io.opentelemetry.instrumentation:opentelemetry-instrumentation-annotations;version='1.19.0.alpha';strategy=exact,\
+	io.opentelemetry.instrumentation:opentelemetry-instrumentation-api;version='1.19.0';strategy=exact,\
+	io.opentelemetry.instrumentation:opentelemetry-instrumentation-api-semconv;version='1.19.0.alpha';strategy=exact,\
+	io.opentelemetry:opentelemetry-api;version='1.19.0';strategy=exact,\
+	io.opentelemetry:opentelemetry-api-logs;version='1.19.0.alpha';strategy=exact,\
+	io.opentelemetry:opentelemetry-context;version='1.19.0';strategy=exact,\
+	io.opentelemetry:opentelemetry-exporter-jaeger;version='1.19.0';strategy=exact,\
+	io.opentelemetry:opentelemetry-exporter-logging;version='1.19.0';strategy=exact,\
+	io.opentelemetry:opentelemetry-exporter-otlp-common;version='1.19.0';strategy=exact,\
+	io.opentelemetry:opentelemetry-exporter-otlp;version='1.19.0';strategy=exact,\
+	io.opentelemetry:opentelemetry-exporter-common;version='1.19.0';strategy=exact,\
+	io.opentelemetry:opentelemetry-exporter-zipkin;version='1.19.0';strategy=exact,\
+	io.opentelemetry:opentelemetry-extension-trace-propagators;version='1.19.0';strategy=exact,\
+	io.opentelemetry:opentelemetry-sdk;version='1.19.0';strategy=exact,\
+	io.opentelemetry:opentelemetry-sdk-common;version='1.19.0';strategy=exact,\
+	io.opentelemetry:opentelemetry-sdk-extension-autoconfigure-spi;version='1.19.0';strategy=exact,\
+	io.opentelemetry:opentelemetry-sdk-extension-autoconfigure;version='1.19.0.alpha';strategy=exact,\
+	io.opentelemetry:opentelemetry-sdk-logs;version='1.19.0.alpha';strategy=exact,\
+	io.opentelemetry:opentelemetry-sdk-metrics;version='1.19.0';strategy=exact,\
+	io.opentelemetry:opentelemetry-sdk-trace;version='1.19.0';strategy=exact,\
+	io.opentelemetry:opentelemetry-semconv;version='1.19.0.alpha';strategy=exact,\
```

Comment:
Unsure that this is necessary, I think `-buildpath` should pick the lowest available matching version.

Reply:
I remember we discussed this and decided using exact here was not required but was preferable.

In the future we may add newer versions of these dependencies and we should get an error if the older versions are removed.